### PR TITLE
chore: add new coys to the adopters page

### PR DESCRIPTION
**Pull request by hacktivist123**
This PR add the following coys to the adopters page:

- JYSK
- SI Analytics
- VK Cloud

### CODE REVIEW

File: src/utils/case-studies-data.js [hunk 1]
```diff
@@ -747,6 +747,32 @@ export default {
...
+      {
+        linkText: 'Watch Talk',
+        linkUrl: 'https://www.youtube.com/watch?v=udPCzFEuDJ0',
+        linkTarget: '_blank',
+      },
+    ],
+  },
+  jsyk: {
+    iconName: 'jysk',
+    text: '<b>JSYK</b> uses Cilium and Hubble in their Kubernetes Clusters for advanced networking, security and observability',
```
Comment (xmulligan) [inline src/utils/case-studies-data.js:763]:
```suggestion
    text: '<b>JSYK</b> uses Cilium and Hubble in their Kubernetes edge clusters for advanced networking, security, and observability',
```

**Comment by xmulligan** [comment]
Can you also add VK to the "Deploy on your preferred cloud" section too?

File: src/utils/case-studies-data.js (hunk 1 above)
Comment (hacktivist123) [reply src/utils/case-studies-data.js]: updated

**Comment by hacktivist123** [comment]
@xmulligan all done

File: src/utils/case-studies-data.js [hunk 2]
```diff
@@ -747,6 +747,32 @@ export default {
...
+        linkTarget: '_blank',
+      },
+    ],
+  },
+  jsyk: {
+    iconName: 'jysk',
+    text: '<b>JSYK</b> uses Cilium and Hubble in their Kubernetes edge Clusters for advanced networking, security and observability',
+    links: [
+      {
+        linkText: 'Watch Talk',
```
[the remark this answers is not in the document]
Comment (hacktivist123) [reply src/utils/case-studies-data.js:766]: oh yeah, my bad will fix it.

**Review by xmulligan — APPROVED** [review]

**Pull request merged**